gradle.properties

```diff
@@ -17,4 +17,6 @@ POM_DEVELOPER_ID=vanniktech
 POM_DEVELOPER_NAME=Niklas Baudy
 
 android.enableR8=true
-android.suppressUnsupportedOptionWarnings=android.suppressUnsupportedOptionWarnings,android.enableR8
\ No newline at end of file
+android.suppressUnsupportedOptionWarnings=android.suppressUnsupportedOptionWarnings,android.enableR8
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
What do we need this one for?

Reply:
just looked at [the docs](https://developer.android.com/jetpack/androidx/migrate); apparently if there are no 3rd-party deps, this flag can be `false`. Today I learned!

Would you prefer it absent, or set to `false`?

Reply:
We shouldn't declare it. Do we need to declare AndroidX if everything that we do is AndroidX?

Reply:
okay, I'll remove the Jetifier line. 

However, according to Xav, the AndroidX line is still necessary as a signal to Android Studio, even if all the deps are AndroidX. Apparently it's used for templates and deps that AS auto-resolves.